blocks/hero/__title/hero__title.css

```diff
@@ -0,0 +1,26 @@
+.hero__title{
+    justify-content: center;
+    color: rgb(255, 255, 255);
+    text-align: center;
+    font-size: 50px;
+    font-weight: 100;
+    line-height: 1.4;
+    margin: 0;
+}
+
+@media(max-width:1199px){
+    .hero__title{
+        font-size: 40px;
+        font-weight: 300;
+    }
+}
+
+@media(max-width:659px){
+    .hero__title{
+        font-size: 24px;
+
+    }
+}
+
+
+/* font-size: 24px; */
```

Comment:
@NazarPauk please remove any accidental comments and empty lines ;)

Reply:
Done✅